src/cleanup_rules/swift/rules.toml

```diff
@@ -890,6 +890,153 @@ not_contains = [
     )""",
 ]
 
+[[rules]]
+name = "de_parenthesize_literals"
+query = """(
+    (tuple_expression
+        (_) @tvalue
```

Comment:
let's only restrict this to literals 
Can we also have some documentation (before/after example) for this rule ?